frontends/mit-learn/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -296,15 +307,15 @@ const SlickCarousel = () => {
 const TestimonialsSection: React.FC = () => {
   return (
     <Section>
-      <Container id="hamster-noises">
+      <HeaderContainer id="hamster-noises">
         <Typography component="h2" variant="h2">
```

Comment:
Looking at Figma, this should be 
```suggestion
        <Typography component="h2" typography={{xs: "h3", sm: "h2"}}>
```